Elevel/Elevel.Api/appsettings.json

```diff
@@ -9,7 +9,7 @@
 
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection": "Server=localhost\\SQLEXPRESS03;Database=Elevel;Trusted_Connection=True;MultipleActiveResultSets=true",
+    "DefaultConnection": "Server=localhost\\SQLEXPRESS;Database=Elevel;Trusted_Connection=True;MultipleActiveResultSets=true",
```

Comment:
The first variant (local) is quite universal, so you needn't to change anything there yet. BTW, because of this we have a conflict, so please accept the (local) variant